frontend/src/app/shared/multi-tools/multi-tools.component.html

```diff
@@ -6,14 +6,5 @@
           (isMultiMode() ? exitLabel :
                            enterLabel) | translate"
   (iconClick)="toggleMultiMode()"></app-icon-button>
-<app-icon-button
-  [icon]="multiActionIcon"
-  class="multi-action"
-  [disabled]="disabledPredicate()"
-  [ngClass]="{ 'active': isMultiMode() }"
-  [label]="
-          isMultiMode() ?
-            ((disabledPredicate() ? multiActionLabelDisabled :
-                                    multiActionLabel) | translate
-            ): ''"
-  (iconClick)="multiActionDelegate()"></app-icon-button>
+<ng-content select="app-multi-action-button"></ng-content>
```

Comment:
Do you prefer this to passing disabledPredicate and isMutiMode to the multi-action-button child/children? 

I just think about the ability for the MultiActionButton to be  a standalone component, when you explicity say that disabledPredicate and isMultiMode will not be undefined but they depend on the inputs from the multi-tools component parent.

Reply:
It's not just preferable, this is entirely by design. The way it's implemented, passing those properties separately would only mean code repetition, since it makes no sense to differentiate them from the parent in any case.
In fact it _should_ use the corresponding properties, which this pattern ensures.

The `MultiActionButton` isn't meant to be used outside the `MultiTools` component.